packages/currency/src/erc20/networks/optimism.ts

```diff
@@ -0,0 +1,20 @@
+import { TokenMap } from './types';
+
+// List of the supported bsc network tokens
+export const supportedOptimismERC20: TokenMap = {
```

Comment:
do we do that for other networks? it seems unnecessary

Reply:
We do yes

Reply:
It's used for the currency default list. So not mandatory, but I think it's good to have some currencies for each network listed there